Dockerfile-alpine.template

```diff
@@ -3,6 +3,8 @@ FROM alpine:3.4
 ENV NPM_CONFIG_LOGLEVEL info
 ENV NODE_VERSION 0.0.0
 
+COPY patches
```

Comment:
the COPY instruction is only needed for alpine on ppc64le. Since alpine is using MUSL libc but not gcc libc. the `sampler.cc` in v8 needs to be patched. I removed these patch files when the patch is applied.